exercises/pangram/example.go

```diff
@@ -0,0 +1,32 @@
+package pangram
+
+import (
+	"strings"
+)
+
+const testVersion = 1
+
+func is_pangram(s string) bool {
```

Comment:
Go uses MixedCaps for names: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps